### ISSUE TITLE
Translation of Mod Menu for AppleSkin mod through the resource pack does not work.

### ISSUE THREAD

**Issue by Disguys** [squeek502/AppleSkin#208]
Hello, I have a problem. For some reason, my ModMenu translation for the AppleSkin mod is not applied through my resource pack.
Everything works fine with other mods, but for some reason will not apply to the AppleSkin mod.

You can use this resource pack to make sure that the translation does not work.
[ModMenu AppleSkin.zip](https://github.com/squeek502/AppleSkin/files/9457212/ModMenu.AppleSkin.zip)

**Comment by Disguys** [comment]
I used the following type of structure for translation
```
{"modmenu.summaryTranslation.appleskin":"§d● §7Shows how much this or that food will satisfy you.","modmenu.descriptionTranslation.appleskin":"§d● §7Shows how much this or that food will satisfy you."}

**Comment by squeek502** [comment]
Unsure what would cause this.

**Comment by squeek502** [comment]
I think what's happening is that AppleSkin provides these strings in our lang files:

https://github.com/squeek502/AppleSkin/blob/2f311872f5be7ef96023ea2c71314709feadcaed/resources/assets/appleskin/lang/en_us.json#L14-L15

and for some reason the resource pack's strings are not taking precedence. I'm not sure exactly how the game chooses which string to prioritize.

**Comment by Disguys** [comment]
Strangely, other mods like iris also have a translation of these lines, but the resource pack can change their description.

**Comment by squeek502** [comment]
Yeah, I tried comparing with another mod that also includes those language strings and wasn't able to figure out why the resource pack worked for the other mod but not for AppleSkin.

**Comment by Disguys** [comment]
I checked it on Fabric 1.18.2 and no similar problem was found there.
